.gitlab-ci.yml

```diff
@@ -32,7 +32,7 @@ tests:
     strategy: depend
   parallel:
     matrix:
-      - GHC_VERSION: [8.8.4, 8.10.7, 9.2.5, 9.4.3]
+      - GHC_VERSION: [8.8.4, 8.10.7, 9.2.5, 9.4.3, 9.6.1]
```

Comment:
This change should also happen on GitHub Action. I think we also forgot for 9.4.3?

Reply:
There’s no stackage nightly yet for GHC 9.6, so no GHA for 9.6